blockchain/blockchain.go

```diff
@@ -622,6 +651,21 @@ func stateUpdateByHash(txn db.Transaction, hash *felt.Felt) (*core.StateUpdate,
 	})
 }
 
+func l1MsgHashes(txn db.Transaction, l1HandlerMsgHash *common.Hash) (*felt.Felt, error) {
```

Comment:
By `Hashes` in name did you mean `common.Hash` and `l1HandlerTxnHash` ?

Reply:
I'll update the name to `l1HandlerTxnHashByMsgHash` so that it's hopefully a bit more clear. It takes in an l1 handler msg hash (common.Hash) and returns the corresponding l1 handler transaction hash (felt)